src/Sulu/Bundle/AdminBundle/Navigation/NavigationItem.php

```diff
@@ -282,12 +288,28 @@ public function getDisabled()
     }
 
     /**
+     * @return null|string
+     */
+    public function getDataNavigationUrl()
+    {
+        return $this->dataNavigationUrl;
+    }
+
+    /**
+     * @param string $dataNavigationUrl
+     */
+    public function setDataNavigationUrl($dataNavigationUrl)
+    {
+        $this->dataNavigationUrl = $dataNavigationUrl;
+    }
+
+    /**
      * Returns a copy of this navigation item without its children
      * @return NavigationItem
      */
     public function copyChildless()
     {
-        $new = new NavigationItem($this->getName());
+        $new = $this->newInstance();
```

Comment:
@danrot you have implemented this functionality first which has confused me and i tried to make it work with inheritance ... but i think clone is not a solution, renaming would propably make it more clean! but i cannot of a good name? what did you think?